operator/.bingo/promtool.mod

```diff
@@ -0,0 +1,37 @@
+module _ // Auto generated by https://github.com/bwplotka/bingo. DO NOT EDIT
+
+go 1.16
+
+replace k8s.io/klog => github.com/simonpasquier/klog-gokit v0.3.0
+
+replace k8s.io/klog/v2 => github.com/simonpasquier/klog-gokit/v2 v2.0.1
+
+exclude github.com/grpc-ecosystem/grpc-gateway v1.14.7
+
+exclude google.golang.org/api v0.30.0
+
+exclude (
+	k8s.io/client-go v1.4.0
+	k8s.io/client-go v1.4.0+incompatible
+	k8s.io/client-go v1.5.0
+	k8s.io/client-go v1.5.0+incompatible
+	k8s.io/client-go v1.5.1
+	k8s.io/client-go v1.5.1+incompatible
+	k8s.io/client-go v10.0.0+incompatible
+	k8s.io/client-go v11.0.0+incompatible
+	k8s.io/client-go v2.0.0+incompatible
+	k8s.io/client-go v2.0.0-alpha.1+incompatible
+	k8s.io/client-go v3.0.0+incompatible
+	k8s.io/client-go v3.0.0-beta.0+incompatible
+	k8s.io/client-go v4.0.0+incompatible
+	k8s.io/client-go v4.0.0-beta.0+incompatible
+	k8s.io/client-go v5.0.0+incompatible
+	k8s.io/client-go v5.0.1+incompatible
+	k8s.io/client-go v6.0.0+incompatible
+	k8s.io/client-go v7.0.0+incompatible
+	k8s.io/client-go v8.0.0+incompatible
+	k8s.io/client-go v9.0.0+incompatible
+	k8s.io/client-go v9.0.0-invalid+incompatible
+)
+
+require github.com/prometheus/prometheus v1.8.2-0.20210220213500-8c8de46003d1 // cmd/promtool
```

Comment:
I checked prometheus releases and the latest is tagged `v2.33.3` which corresponds to commit `56e14463bccfbb6a8facfb663fed5e0ca9f8b387`
https://github.com/prometheus/prometheus/releases/tag/v2.33.3

So I ran the following command to update promtool
```
bingo get -v github.com/prometheus/prometheus/cmd/promtool@56e14463bccfbb6a8facfb663fed5e0ca9f8b387
```
I think that it did the trick, but the updated binary still has `v1.8.2` in its file name: `promtool-v1.8.2-0.20220211202545-56e14463bccf`

Reply:
Yes, unfortunately the binary will always have the wrong version in the name because the repository layout does not match the requirements of Go modules. This should be the correct version, though.